client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -437,8 +436,8 @@ public void testStartDatafeed() throws Exception {
         highLevelClient().indices().create(createIndexRequest, RequestOptions.DEFAULT);
         BulkRequest bulk = new BulkRequest();
         bulk.setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE);
-        long now = System.currentTimeMillis();
-        long oneDayAgo = now - 86400000;
+        long now = (System.currentTimeMillis()/1000)*1000;
```

Comment:
Are you rounding down to the nearest second here? Do you not need an `L` to ensure the result of `System.currentTimeMillis()/1000` is an integer i.e. System.currentTimeMillis()/1000L

Reply:
yeah, `oneDayAgo` should just be renamed as that is not what it is any more